distribution/docs/src/main/resources/_contents/_draft-catalog-contents/managing-catalog-contents.adoc

```diff
@@ -0,0 +1,47 @@
+
+The ${ddf-catalog} provides a framework for storing, searching, processing, and transforming information.
+
+Clients typically perform query, create, read, update, and delete (QCRUD) operations against the Catalog.
```

Comment:
I hadn't either. I can simplify, as I don't think we make this distinction anywhere else. (If there even is a distinction)